test/extension-developer-e2e/setup.sh

```diff
@@ -75,11 +75,14 @@ reg_pkg_name="${REG_PKG_NAME}"
 (
   cd "${REG_DIR}" && \
   $operator_sdk init --domain="${DOMAIN}" && \
+  sed -i -e 's/CONTROLLER_TOOLS_VERSION ?= v0.14.0/CONTROLLER_TOOLS_VERSION ?= v0.15.0/' Makefile && \
```

Comment:
Delete this line? Also while we're at it, could you add some comments about this `sed` command and the `export OPERATOR_SDK` line?

Something like these:
-  `# controller-gen v0.13.0 (scaffolded by operator-sdk) panics when run with go 1.22, so pin to a more recent version. NOTE this is a rough edge that users will experience`
- `# The Makefile in the project scaffolded by operator-sdk uses an SDK binary in the path if it is present. Override to ensure we use the same version that we scaffolded with. NOTE: this is a rough edge that users will experience`

Reply:
(sorry, didn't see/think of this when we we're debugging it)

Reply:
I'll make sure it still works without this...